.github/workflows/reusable-test.yml

```diff
@@ -49,15 +57,15 @@ jobs:
           name: Test results
           path: '*.xml'
           reporter: java-junit
-      - name: Report coverage
-        # noinspection SpellCheckingInspection
-        uses: livewing/lcov-job-summary@0a9952db27f98def2ba2de3189c6b24acc4c5501 # v1.1.0
-        if: success() || failure()
+      - name: Upload coverage results
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4
         with:
-          lcov: coverage/davidlj95/website/lcov.info
```

Comment:
Haven't checked those since I put them there 😛  Sooo letting that go in favour of Codecov